applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -25,18 +26,55 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_LOG_LEVEL);
 #define FLASH_PAGE_SIZE_LOG2	12
 #define FLASH_PAGE_SIZE		BIT(FLASH_PAGE_SIZE_LOG2)
 #define FLASH_PAGE_ID(off)	((off) >> FLASH_PAGE_SIZE_LOG2)
+#define FLASH_CLEAN_VAL		UINT32_MAX
+#define FLASH_READ_CHUNK_SIZE	(FLASH_PAGE_SIZE / 8)
```

Comment:
identation here or line obove?

Reply:
Github shows it in strange way, it is correct